### DISCUSSION TITLE
Add new custom clients automatically to address book

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk#8650] [General]
Hi,

all other remote support systems I use (ISL, Anydesk) allow to automatically add new custom clients (licensed version) to a predefined address book. I really miss this feature. I roll out clients by device management and clients should be visible in the address book without adding them manually.

I've tried "Sync with recent sessions" as a workaround without success. But I also don't know what "Sync with recent sessions" really means ...

It would also be nice to be able to set a default address book for a authenticated user.

Matthias

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#preset-address-book-name--preset-address-book-tag

**Reply by mahescho** [reply]
Works as expected, thanks.
